docs/copied-from-beats/docs/security/users.asciidoc

```diff
@@ -18,9 +18,10 @@ by {beatname_uc}
 create visualizations that access {beatname_uc} data
 
```

Comment:
If there's one thing you look at in this PR, make it this page. There's just so much going on here, it's tough to see if I caught everything that needs to be changed for APM Server. Here's a direct link to the preview for this page: http://apm-server_3227.docs-preview.app.elstc.co/guide/en/apm/server/master/feature-roles.html.